packages/actiony/db/migrations/1671032411845-InitialMigration.ts

```diff
@@ -0,0 +1,40 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class InitialMigration1671032411845 implements MigrationInterface {
+  public name = 'InitialMigration1671032411845';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`
+            CREATE TYPE "actiony"."action_action_status_enum" AS ENUM('active', 'completed', 'failed', 'canceled')
+        `);
+    await queryRunner.query(`
+            CREATE TABLE "actiony"."action" (
+                "action_id" uuid NOT NULL DEFAULT uuid_generate_v4(),
+                "service" character varying NOT NULL,
+                "state" integer NOT NULL,
+                "rotation" character varying NOT NULL,
```

Comment:
Maybe it is not best to save "1.3" to the database, what about rotation Major and Minor that combines the string.

instead of a character column, two long number columns.

wdyt?

Reply:
done.